app/_src/policies/meshtls.md

```diff
@@ -0,0 +1,85 @@
+---
+title: MeshTLS
+---
```

Comment:
Should we add a guide along with this named something like: "Progressively rolling in strict MTLS" and then we could link it from this page?

Something that's not clear is how clients get influenced by the setting of MeshTLS. We had a discussion about this earlier this week, we should really document how this works.

Reply:
created a separate issue for this https://github.com/kumahq/kuma-website/issues/1897

Reply:
> Something that's not clear is how clients get influenced by the setting of MeshTLS. We had a discussion about this earlier this week, we should really document how this works.

In line https://github.com/kumahq/kuma-website/pull/1893/files#diff-9149b3f0d9714aeffecfdede60cbc8469e9593e7acee44002e0d0aa61444b201R35 I'm mentioning that we configure both server and client but I'll make it more explicit.

Reply:
added something like this: https://github.com/kumahq/kuma-website/pull/1893/files#diff-9149b3f0d9714aeffecfdede60cbc8469e9593e7acee44002e0d0aa61444b201R40